### PR TITLE
Remove redundant check

### PR DESCRIPTION
**Pull request by polaris6**
## Description
Remove redundant check

## Documentation
(x) No documentation is needed.

## Release notes
(x) No release notes entries required.

### CODE REVIEW

**Comment by findepi** [comment]
Can you expand commit message and elaborate why the check is redundant?
It's best if you split this into 3 commits.
(while it looks like an overkill, it's not. The change wasn't obvious to me.)

**Comment by polaris6** [comment]
Hi @findepi, thanks for your advice, I updated the commits.

**Review by findepi — DISMISSED** [review]

File: plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraSession.java [hunk 1]
```diff
@@ -395,7 +395,7 @@ public List<CassandraPartition> getPartitions(CassandraTable table, List<Set<Obj
             rows = queryPartitionKeysLegacyWithMultipleQueries(table, filterPrefixes);
         }
 
-        if (rows == null) {
+        if (rows.isEmpty()) {
             // just split the whole partition range
             return ImmutableList.of(CassandraPartition.UNPARTITIONED);
```
Comment (findepi) [inline plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraSession.java:400]: This used to be a dead code. 
So the correct "naive update" would be to remove the block.
Is this codepath exercised? 

cc @ebyhr

Comment (ebyhr) [reply plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraSession.java:400]: I think it's a dead code. Let's remove that.

Comment (polaris6) [reply plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraSession.java]: Thanks for your advice, I updated the commits.

Comment (polaris6) [reply plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraSession.java]: There is a failing check, but it's not caused by this PR and can be ignored.

**Review by findepi — APPROVED** [review]

**Pull request merged**